openslides_backend/action/actions/meeting/import_.py

```diff
@@ -457,11 +511,11 @@ def create_write_requests(
         write_requests = []
         for collection in json_data:
             for entry in json_data[collection].values():
-                entry.pop("meta_new", None)
+                meta_new = entry.pop("meta_new", None)
                 fqid = FullQualifiedId(Collection(collection), entry["id"])
                 write_requests.append(
                     self.build_write_request(
-                        EventType.Create,
+                        EventType.Create if meta_new else EventType.Update,
```

Comment:
The place to use also parameter list_fields

Reply:
**Cautions: entry["id"] != key.**

Reply:
Outdated